src/docs/product/profiling/getting-started.mdx

```diff
@@ -28,3 +28,4 @@ Profiling depends on Sentry's performance monitoring product being enabled befor
   - [iOS](/platforms/apple/guides/ios/profiling/)
   - [Python](/platforms/python/profiling/)
   - [Node.js](/platforms/node/profiling)
+  - [Rust](/platforms/rust/profiling)
```

Comment:
Is there any reason to not start putting these in alphabetical order?

Reply:
no, good point. I'm fixing that